modules/http/http_filters.c

```diff
@@ -303,6 +309,122 @@ static apr_status_t read_chunked_trailers(http_ctx_t *ctx, ap_filter_t *f,
 apr_status_t ap_http_filter(ap_filter_t *f, apr_bucket_brigade *b,
                             ap_input_mode_t mode, apr_read_type_e block,
                             apr_off_t readbytes)
+{
```

Comment:
What follows is the code that used to be at the end of the function. Diff just shows them as addition here and removal below.